code/SecurityAdmin.php

```diff
@@ -246,8 +274,14 @@ public function MemberImportForm()
         return $form;
     }
 
+    /**
+     * @deprecated 1.13.0 Will be removed without equivalent functionality to replace it
```

Comment:
In hindsight these should probably say "without equivalent API" not functionality, because there _is_ equivalent functionality, just not with an API like this.
Too late to change now though.